LegStamped A_LF
LegStamped B_RF
LegStamped C_RH
LegStamped D_LH
geometry_msgs/Pose pose
geometry_msgs/Twist twist